content/en/blog/_posts/XX-dra-consumable-capacity.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: blog
+title: "Kubernetes v1.34: DRA Consumable Capacity"
+draft: true # will be changed to date: YYYY-MM-DD before publication
+slug: kubernetes-v1-34-dra-consumable-capacity # optional
+author: >
+  Sunyanan Choochotkaew (IBM),
+  Lionel Jouin (Ericsson Software Technology)
+  John Belamaric (Google)
+---
+
+# Kubernetes v1.34: DRA Consumable Capacity
+
+Dynamic Resource Allocation (DRA) is a Kubernetes API for managing scarce resources across Pods and containers.
+It enables flexible resource requests, going beyond simply allocating *N* number of device to support more granular usage scenarios.
+With DRA, users can request specific types of devices based on their attributes, define custom configurations tailored to their workloads, and even share the same resource among multiple containers or Pods.
+
+In this blog, we focus on the device sharing feature and dive into a new capability introduced in Kubernetes 1.34: `DRAConsumableCapacity`, which extends DRA to support finer-grained device sharing.
+
+## Background: Device Sharing via ResourceClaims
+
+From the beginning, DRA introduced the ability for multiple Pods to share a device by referencing the same ResourceClaim.
+This design decouples resource allocation from specific hardware, allowing for more dynamic and reusable provisioning of devices.
+
+In Kubernetes 1.33, the Partitionable Devices feature added support for resource drivers to advertise slices of a device that are available—rather than exposing the entire device as an all-or-nothing resource.
```

Comment:
```suggestion
In Kubernetes 1.33, the Partitionable Devices feature added support for resource drivers to advertise slices of a device that are available, rather than exposing the entire device as an all-or-nothing resource.
```